companies/verdi-de.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Missed this on the previous ones, but all records need a slug identical to the filename (without the extension). Otherwise the build will fail.

Reply:
Thanks. I used https://company-json.netlify.com to generate the slug based on the domain. But while it was entered to the slug textbox, it didn't appear in the JSON file.

Reply:
Yeah, that is unfortunately another but of the library we use. :(

I implemented a bunch of useful features only to notice that brutusin-json-forms doesn't notice the changes by these features unfortunately.